cdci_data_analysis/flask_app/dispatcher_query.py

```diff
@@ -471,7 +471,11 @@ def generate_job_id(self, kw_black_list=None):
         self.logger.info(
             "\033[31m---> new job id for %s <---\033[0m", self.par_dic)
 
-        self.logger.debug("generate_job_id: %s", json.dumps(self.par_dic, indent=4, sort_keys=True))
+        try:
+            self.logger.debug("generate_job_id: %s", json.dumps(self.par_dic, indent=4, sort_keys=True))
+        except Exception as e:
+            self.logger.error("unable to jsonify this self.par_dic = %s", self.par_dic)
+            raise
```

Comment:
I have had sometimes problems here, but was always related to some entries inside `par_dic` not properly formatted.

Reply:
That's just for more visibility. I experienced it too. Bit it changes nothing of course.